ytdlp_handler.py

```diff
@@ -73,11 +91,12 @@ def _gen_query(
         "noplaylist": True,
```

Comment:
Rather than playing with the index in playlist item if the playlist box is or isn't ticked you should set this option to the value of the box

Reply:
It's seems not working alone.

If I put a value like this :

```python
options = {
    "noplaylist": not playlist,
    #"playlist_items": None if playlist else "1",
}
```

The first playlist video download (if playlist is not ticked) doesn't work.
I use `not playlist` but I also use "playlist_items".

Do you have another idea ?